database/dbseeding/fakerAlteration.js

```diff
@@ -0,0 +1,135 @@
+/* eslint-disable */
+
+const pathToCompany = 'faker/lib/company.js';
+this.companyName = function (format) {
+  const adjective = ["Organic", "Fresh", "Family", "Super", "Giant", "BigOl", "Tasty", "Juicy", "Vegan", "Big", "Delicious"];
+  const cuisine = ["Italian", "Thai", "Indian", "Vietnamese", "Bulgarian", "Chinese", "Mexican", "Russian", "Slavic", "European", "French", "Japanese", "Korean", "German", "British"];
+  const foodSingular = ["Burger", "Sushi", "Pizza", "Casserole", "Chili Bowl", "Soup", "Pie", "Cake", "Burrito", "Taco", "Salad", "Pho", "Chili", "Momo", "StirFry", "FishNChips", "MacNCheese", "Pasta", "Spaghetti"];
+  const foodPlural = ["Burgers", "Burritos", "Country Dumplins", "Grits", "Tacos", "Salads", "Dumplings", "Soups", "Pies", "Cakes", "Smoothies"];
+  const place = ["Bar", "Restaurant", "Buffet", "Hub", "Spot", "Corner", "Avenue", "Street", "Table", "Diner", "Cafe", "Bistro"];
+  const random = (array) => (array[faker.random.number({ min: 0, max: array.length - 1, precision: 1 })]);
+
+  var formats = [
+    `{{name.lastName}}s ${random(cuisine)} ${random(foodPlural)}`,
+    `{{name.lastName}}s`,
+    `${random(foodSingular)} ${random(place)}`,
+    `{{name.firstName}}s ${random(foodSingular)} ${random(place)}`,
+    `The ${random(foodSingular)} ${random(place)}`,
+    `The ${random(adjective)} ${random(foodSingular)}`,
+  ];
+
+  if (typeof format !== "number") {
+    format = faker.random.number(formats.length - 1);
+  }
+
+  return f(formats[format]);
+}
+
+const pathToWords = 'faker/lib/locales/en/lorem/words.js'
+const words = [  "delicious",
```

Comment:
I love this! 🌯

Reply:
I had some fun with it but my creativity eventually wore off lol